src/Aop/Support/ModifierMatcherFilter.php

```diff
@@ -47,6 +47,7 @@ class ModifierMatcherFilter implements PointFilter
     public function __construct($initialMask = 0)
     {
         $this->andMask = $initialMask;
+        $this->notMask = \ReflectionMethod::IS_ABSTRACT;
```

Comment:
@lisachenko Ok, it seams that you are right, modifier filter should have default value of \ReflectionMethod::IS_ABSTRACT for `notMask` .

Test passes, but what is worries me is that there are no a lot of integration/functional tests where we can test what we actually get (like what I did for console commands). 

What do you think about writing some functional tests that would have real-life fixtures for verification of weaving?